pkg/config/config_template.yaml

```diff
@@ -1143,12 +1143,33 @@ api_key:
   ## Defines rules by which to filter traces based on tags.
   ##  * require - list of key or key/value strings - traces must have those tags in order to be sent to Datadog
   ##  * reject - list of key or key/value strings - traces with these tags are dropped by the Agent
-  ## Note: Rules take into account the intersection of tags defined.
+  ## Please note that:
+  ##   1. Rules take into account the intersection of tags defined.
+  ##   2. When `filter_tags` and `filter_tags_regex` are used at the same time, all rules are united for filtering.
+  ##      In cases where rules in `filter_tags` and `filter_tags_regex` have the same key, the rule from `filter_tags`
+  ##      takes precendence over the rule from `filter_tags_regex`.
+  ##
+  ##      In the case of the following configuration:
+  ##        filter_tags:
+  ##          require: ["foo:bar"]
+  ##        filter_tags_regex:
+  ##          require: ["foo:^bar[0-9]{1}$"]
+  ##      The resulting combined rule will match `foo:bar` and not `foo:bar1`
```

Comment:
Rather than saying what it will match, can you instead state the result. e.g. "With these rules, traces with a tag `<insert_tag>` will be dropped, and those with a `<insert_tag>` tag  will be kept" (or something similar)

Reply:
I described in a little more detail what the result will be as a result of filtering span with the specified rules